build.sbt

```diff
@@ -150,10 +151,11 @@ lazy val facade =
       // Compile tests to JS using fast-optimisation
       scalaJSStage in Test := FastOptStage,
       libraryDependencies ++= Seq(
-        "com.github.japgolly.scalajs-react" %%% "core"   % scalaJsReact,
-        "io.github.cquiroz.react"           %%% "common" % scalaJsReactCommon,
-        "com.github.japgolly.scalajs-react" %%% "test"   % scalaJsReact % Test,
-        "com.lihaoyi"                       %%% "utest"  % uTest        % Test
+        "com.github.japgolly.scalajs-react" %%% "core"                 % scalaJsReact,
+        "io.github.cquiroz.react"           %%% "common"               % scalaJsReactCommon,
+        "io.github.toddburnside"            %%% "scalajs-react-window" % scalajsReactWindow,
+        "com.github.japgolly.scalajs-react" %%% "test"                 % scalaJsReact % Test,
+        "com.lihaoyi"                       %%% "utest"                % uTest        % Test
```

Comment:
Would we start using `munit` instead?

Reply:
I started writing some tests and switched to munit, but stopped working on them after our conversation.